docs/application-connector/docs/015-details-one-click-configuration.md

```diff
@@ -14,7 +14,7 @@ Automatic configuration flow is presented in following diagram:
 
 This example assumes that the new Remote Environment already exists and it is in the `disconnected` state, which means that there are no solutions connected to it.
 
-On the diagram, Admin on the Kyma side and on external system side is the same person.
+On the diagram, the admin on the Kyma side and on the external system side is the same person.
```

Comment:
Shouldn't we use "administrator" instead of "admin"? On the diagram, there's also "administrator" in the full form.